src/utils/spawning.rs

```diff
@@ -424,10 +424,9 @@ mod systemd {
 
         trace!("waiting for JobRemoved");
         for message in signals {
-            let body: (u32, OwnedObjectPath, &str, &str) =
-                message.body().context("error parsing signal")?;
+            let header = message.header();
```

Comment:
Is this right? (I'm not sure, it's just weird that body changed to header)

Reply:
You're totally right, I was going through all of the boring changes and wasn't paying enough attention, I looked for a method returning `ObjectPath` not realizing this was deserializing and the path came from the data.

(eghh apparently `Resolve conversation` deletes your comment, some great UX github)